Bottom: 8dfd198b79ef4238753fcfe1490a8a0febc00634
Top:    a9521bf0d4e7b5893744d3cf2e84744f0d5179a5
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-12 13:36:08 +0800

Refresh of tuntap-support-xdp-offloading

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index ad8abbe..ccc7f22 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1377,6 +1377,7 @@ static u32 tun_do_xdp_offload(struct tun_struct *tun, struct xdp_frame *frame)
 		act = bpf_prog_run_xdp(xdp_prog->prog, &xdp);
 		switch (act) {
 		case XDP_TX:
+			/* FIXME! */
 			break;
 		case XDP_PASS:
 			break;
